backend/main_original.py

```
@@ -0,0 +1,468 @@
+                }
+            ],
+            max_tokens=300
+        )
+        log.info("OpenAI API call successful!")
+        log.info(f"Response object type: {type(response)}")
+        log.info(f"Response choices length: {len(response.choices)}")
+        log.info(f"First choice: {response.choices[0]}")
+        
+        content = response.choices[0].message.content.strip()
+        log.info(f"Content type: {type(content)}")
+        return content
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)